great-idea/services.html

```diff
@@ -1,87 +1,123 @@
+                    eleifend. Mauris euismod facilisis iaculis.</p>
 
+                <button>Learn More</button>
+            </div>
 
-Android Development
+            <div class="ios-development services-content-box">
+                <h5>iOS Development</h5>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+                <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.</p>
 
-Learn More
+                <button>Learn More</button>
+            </div>
 
+            <div class="android-development services-content-box">
+                <h5>Android Development</h5>
 
-Some Facts About Our Services
+                <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                    eleifend. Mauris euismod facilisis iaculis.</p>
 
-Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
-eleifend. Mauris euismod facilisis iaculis.
+                <button>Learn More</button>
+            </div>
+        </section>
 
+        <section class="services-bottom-content">
+            <div class="services-bottom-text"> 
+                <h5>Some Facts About Our Services</h5>
 
-Awesome thing
-Amazing thing
-Cool thing
-Great thing
+                <p>Nulla suscipit interdum diam at maximus. Proin vitae tellus rutrum, accumsan nunc sit amet, iaculis tortor. Mauris consectetur feugiat justo quis aliquet. Curabitur cursus et justo eu consequat. Fusce finibus sem neque, sed faucibus nibh pellentesque
+                eleifend. Mauris euismod facilisis iaculis.</p>
 
+                <ul>
+                    <li>Awesome thing</li>
+                    <li>Amazing thing</li>
+                    <li>Cool thing</li>
+                    <li>Great thing</li>
+                </ul>
+            </div> 
 <img class="services-info-img" src="img/services-info.png" alt="one of our employees hard at work">
-
-Copyright Great Idea! 2018
+        </section>
+
+        <footer>
+            <p>Copyright Great Idea! 2018</p>
+        </footer>
+<!-- 
+Copyright Great Idea! 2018 -->
+        </div> <!-- class=container -->
```

Comment:
I like seeing this comment! It will help later on when the code gets longer. I like to do the same in my javascript files when a function or class gets very long.